.circleci/build-default-workloads.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+from fabric.api import *
+
+from common import manager_fsim_dir, manager_hostname
+from ci_variables import ci_workflow_id
+
+def build_default_workloads():
+    """ Builds workloads that will be run on F1 instances as part of CI """
+
+    with prefix('cd {} && source ./env.sh'.format(manager_fsim_dir)), \
+         prefix('cd deploy/workloads'):
+        run("marshal -v build br-base.json")
```

Comment:
You can run `marshal` inside `deploy/workloads`?  I thought you had to be inside `manager_marshal_dir`.

Reply:
Nevermind, I checked and `env.sh` does put marshal on the path.  I've just never run it this way.